doc/downgrade.8.md

```diff
@@ -91,6 +91,17 @@ on the ALA.
 
 > Whether to add packages to IgnorePkg, default is _prompt_.
 
+**\--latest | --oldest**\
+
+> Never prompt for version or location, automatically pick the most up to
+> date/most out of date version. Allows for noninteractive downgrades when
+> used in conjunction with --ignore
+
+**\--cache-preferred**\
+
+> If a package matching the version filter is found in the cache, skip querying
+> ala.
```

Comment:
I would mention the relationship to `--cached-only`, adding something like "this is like `--cached-only` accept it will still try ALA if no package is found."